meta-resin-common/recipes-core/images/resin-image-flasher.bb

```diff
@@ -9,6 +9,8 @@ RESIN_FLAG_FILE = "${RESIN_FLASHER_FLAG_FILE}"
 # Each machine should append this with their specific configuration
 IMAGE_FSTYPES = ""
 
+RESIN_ROOT_FSTYPE = "ext4"
```

Comment:
I think defining it in resin-image would make it clearer and leave the class depend on the definition on the image recipes. Maybe add in the class a check that RESIN_ROOT_FSTYPE was defined when needed and error out otherwise. In this case we mix the definition in the recipe and the class which can get confusing. (NIT)

Reply:
https://github.com/resin-os/meta-resin/blob/edabcbc20e09322db37153d5ee0b3fc9988c9148/meta-resin-common/recipes-core/images/resin-image.bb doesn't define it at all. The class does it. What I propose is to define it in the recipes so it will be clear which one is which. And the class checks that it is defined.

Reply:
That's how it was implemented previously https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/classes/image_types_resin.bbclass#L59 . We can fix in some other PR